packages/core/src/browser/style/tabs.css

```diff
@@ -75,6 +75,10 @@
 | Tabs in the center area (main and bottom)
 |----------------------------------------------------------------------------*/
 
+#theia-main-content-panel .p-Widget.p-DockPanel-widget {
```

Comment:
let's revert this PR, I would need to fix about consequences of such class, it affects each widget in the main area

Reply:
just realized that it was merged long time ago :)

Reply:
@akosyakov yes quite a while ago :see_no_evil: